package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
you have a package-lock.json but are using yarn. should pick one or the other. suggest remove this file and add yarn lockfile

Reply:
https://github.com/swishlabsco/peggy/pull/2/commits/27ca138840aaf3d80be593b2b41a4f052d8059d3